### DISCUSSION TITLE
Updating CentOS 8.2 to 8.4 or 8.5

### DISCUSSION THREAD

**Discussion by LeoPixelStudios** [AlmaLinux/almalinux-deploy#122] [Q&A]
Good day.. I need to update my server to CentOS 8.4 or 8.5 in order to run the AlmaLinux Upgrade. I must be doing something wrong, because the "sed " codes don't seem to do anything.. 

steps takin so far.

- copied and pasted the lines in my ssh program (needed to change the folder's name.) They are no errors when I do that, but when I run the dnf update.. I get the same error as Plesk update fails.. 
- Tried to update the dnf config by adding [repository] then all the lines.. doesn't work..
- Tried to modify 3 repository from Plesk documentation (Appstream, Base and PowerTools).. update starts, but I get this error...
-
Errors during downloading metadata for repository 'PLESK_17_PHP74':
  - Status code: 404 for http://autoinstall.plesk.com/PHP74_17/dist-rpm-CentOS-8-x86_64/repodata/repomd.xml (IP: 2a02:6ea0:c600::13)
Error: Failed to download metadata for repo 'PLESK_17_PHP74': Cannot download re  pomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried

- I tried to add the "find /etc/yum.repos.d -type f -exec sed -i -e '/mirrorlist=http:\/\/..." But I get an error about the -exec

So i'm out of things to try.. And re-installing the server is a bit out of the question, since there is a live moodle platform on it ;)

thx for you help..

Marc.

**Comment by intellijk** [comment]
I am wondering if you ever got around this problem and managed to upgrade to a higher version. My update is not working either. Could someone kindly point me to the resource I can use to get working mirrors?

Update:

After a many hours of fumbling in the dark, I finally managed to upgrade my centos 8.2 to 8.5. I manually changed all the mirrors inside /etc/yum.repos.d to these ones here https://www.centos.org/download/mirrors/. I was almost giving up but I'm glad I didn't.
